tests/openwrt/test_wireless.py

```diff
@@ -199,7 +199,7 @@ def test_parse_multiple_wifi(self):
     option device 'radio0'
     option ifname 'wlan0'
     option mode 'ap'
-    option network 'wlan0'
+    option network 'br_lan wlan0'
```

Comment:
Having a closer look, I think `wlan0` here was kind of the default because the `network` attribute was not defined, but now, since it's implicitly defined , `wlan0` should disappear, also beacuse it doesn't make any sense from the configuration point of view.